### ISSUE TITLE
Hidden figcaption is a violation unless supported by aria-label

### ISSUE THREAD

**Issue by straker** [dequelabs/axe-core#1581]
This should be a violation

```html
<figure>
  <figcaption hidden>Text</figcaption>
  <img src="..." />
</figure>
```

**Comment by dan-tripp** [comment]
Interesting issue.  I was looking into it just now.  In your desired behaviour @straker , should that sample HTML be a violation under an _existing_ rule?  If so, which rule?
